src/injections/composeInjections/index.js

```diff
@@ -1,5 +1,11 @@
 import mergeInjections from '../mergeInjections';
 
+const checkPaginationNotHasFinished = (state, pageSelector) =>
```

Comment:
This name is a little bit confusing. Would you like to name it using the positive case ?